bucket/gittyup.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "1.1.0",
+    "description": "A continuation of GitAhead client, a graphical Git client designed to help you understand and manage your source code history.",
+    "homepage": "https://murmele.github.io/Gittyup/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Murmele/Gittyup/releases/download/stable/Gittyup-win64-1.1.0.exe#/dl.7z",
+            "hash": "596270fa68ebff7e5a920d69800693b32ae033fd32f496f3f16b0be95de016aa"
+        },
+        "32bit": {
+            "url": "https://github.com/Murmele/Gittyup/releases/download/stable/Gittyup-win32-1.1.0.exe#/dl.7z",
+            "hash": "571ccd71ba5881b81206ec9e03d6a99d6b6d21850963976b1a34ac1f9f1fa359"
+        }
+    },
+    "checkver": {
+        "url": "https://api.github.com/repos/Murmele/Gittyup/releases/latest",
```

Comment:
```suggestion
        "github": "https://github.com/repos/Murmele/Gittyup",
```

Reply:
I do not really understand, is it no better to use github's API like you suggested earlier?

Reply:
Sorry I was wrong on this one. `(1) The latest release of Gittyup is the dev version, which is not what we want.`

If you go to https://github.com/Murmele/Gittyup/releases/latest, it will lead to **Release version 1.1.1**, so using `"github":  "https://github.com/repos/Murmele/Gittyup"` won't be a problem. We only need to add `"regex"` here.

Reply:
Oh, got it!